sdk/tests/conformance/resources/gl-object-get-calls.js

```diff
@@ -170,17 +217,67 @@ var validArrayForFramebufferAttachment = new Array(
 );
 if (contextVersion > 1) {
   for (var ii = 1; ii < gl.getParameter(gl.MAX_COLOR_ATTACHMENTS); ++ii) {
-    validArrayForFramebufferAttachment[validArrayForFramebufferAttachment.length] = gl.COLOR_ATTACHMENT0 + ii;
+    validAttachmentsForFBAttachment[validAttachmentsForFBAttachment.length] = gl.COLOR_ATTACHMENT0 + ii;
   }
 }
 testInvalidArgument(
     "getFramebufferAttachmentParameter",
     "attachment",
-    validArrayForFramebufferAttachment,
+    validAttachmentsForFBAttachment,
     function(attachment) {
       return gl.getFramebufferAttachmentParameter(gl.FRAMEBUFFER, attachment, gl.FRAMEBUFFER_ATTACHMENT_OBJECT_TYPE);
     }
 );
+if (contextVersion > 1) {
+  // test default framebuffer
+  gl.bindFramebuffer(gl.FRAMEBUFFER, 0);
```

Comment:
Accrodingly to ES 3.0 spec, getFramebufferAttachmentParameter can query the information of default framebuffer. But firefox nightly build failed to handle this.